packages/databases-collections-list/src/databases.tsx

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { spacing } from '@mongodb-js/compass-components';
+import { compactBytes, compactNumber } from './format';
+import { NamespaceItemCard } from './namespace-card';
+import { ItemsGrid } from './items-grid';
+
+type Database = {
+  _id: string;
+  name: string;
+  status: string;
+  statusError: string | null;
+  collectionsStatus: string;
+  collectionsStatusError: string | null;
+  collection_count: number;
+  document_count: number;
+  storage_size: number;
+  data_size: number;
+  index_count: number;
+  index_size: number;
+  collectionsLength: number;
+};
+
+const DATABASE_CARD_WIDTH = spacing[6] * 4;
+
+const DATABASE_CARD_HEIGHT = 154;
+
+const DATABASE_CARD_LIST_HEIGHT = 118;
+
+const DatabasesList: React.FunctionComponent<{
+  databases: Database[];
+  onDatabaseClick(id: string): void;
+  onDeleteDatabaseClick?: (id: string) => void;
+  onCreateDatabaseClick?: () => void;
+}> = ({
+  databases,
+  onDatabaseClick,
+  onCreateDatabaseClick,
+  onDeleteDatabaseClick,
+}) => {
+  return (
+    <ItemsGrid
+      items={databases}
+      itemType="database"
+      itemGridWidth={DATABASE_CARD_WIDTH}
+      itemGridHeight={DATABASE_CARD_HEIGHT}
+      itemListHeight={DATABASE_CARD_LIST_HEIGHT}
+      sortBy={[
+        { name: 'name', label: 'Database Name' },
+        { name: 'storage_size', label: 'Storage size' },
+        { name: 'collectionsLength', label: 'Collections' },
+        { name: 'index_count', label: 'Indexes' },
+      ]}
+      onItemClick={onDatabaseClick}
+      onDeleteItemClick={onDeleteDatabaseClick}
+      onCreateItemClick={onCreateDatabaseClick}
+      renderItem={({
+        item: db,
+        onItemClick,
+        onDeleteItemClick,
+        viewType,
+        ...props
+      }) => {
+        return (
+          <NamespaceItemCard
+            id={db._id}
+            key={db._id}
+            name={db.name}
+            type="database"
+            viewType={viewType}
+            status={db.status}
+            data={[
+              {
+                label: 'Storage size',
+                value: compactBytes(db.storage_size),
+                hint: `Uncompressed data size: ${compactBytes(db.data_size)}`,
```

Comment:
Maybe I'm interacting with this in a weird way, but I'd find myself sometimes moving my cursor upwards to the database name. This hint opens even if the cursor movement is pretty quick. Maybe not a real issue, but is there a way we can make this popup not stay open when the cursor is over it, or another way to show this information?
Here's a gif example:

https://user-images.githubusercontent.com/1791149/146313358-bd34ee3d-eae8-44a4-b7c4-91285ccb3fd4.mp4

Reply:
I'll ask Claudia what does she thinks we can do here, I think with these `InlineDefinition` lg components there is no way to make them not appear on hover and leafygreen tooltips don't do any delay before first appearance (as e.g., built-in browser titles do)

Reply:
Hmmm, actually [aria spec](https://www.w3.org/TR/wai-aria-1.1/#tooltip) says that there should be a delay before showing a tooltip. I'll ask in leafygreen channel if they are aware and will be open to fixing it

Reply:
Apparently [that's a known issue](https://jira.mongodb.org/browse/PD-1533), we can try to address it on our side though

Reply:
Done in eb1f332e1aaead42f0d822584bbd60c7de079865